src/test/java/io/strimzi/kafka/bridge/http/HttpBridgeTest.java

```diff
@@ -2876,13 +2963,14 @@ void sendReceiveJsonMessage(VertxTestContext context) throws InterruptedExceptio
 
         CompletableFuture<Boolean> consume = new CompletableFuture<>();
         // consume records
-        client.get(BRIDGE_PORT, BRIDGE_HOST, baseUri + "/records" + "?timeout=" + String.valueOf(1000))
+        getRequest(baseUri + "/records" + "?timeout=" + String.valueOf(1000))
```

Comment:
Why concat two adjacent string literals?
```suggestion
        getRequest(baseUri + "/records?timeout=" + 1000)
```

Reply:
I'm not an author of concatenating for two adjacent String literals. I just updated method name here `client.get(...)` -> `getRequest(...)` Maybe we need it for something. @ppatierno @stanlyDoge are you agree to merge these two Strings.

Reply:
For sure you can remove that `String.valueOf` :-)